lib/fog/local.rb

```diff
@@ -1,16 +1,16 @@
 require 'fog/core'
 require 'fileutils'
 require 'tempfile'
-require File.expand_path('../local/version', __FILE__)
 
-module Fog
-  module Storage
-    autoload :Local, File.expand_path('../storage/local', __FILE__)
-  end
+require 'fog/local/version'
+require 'fog/local/storage'
```

Comment:
Prefer the `autoload` version please. =D

Reply:
We can do that, but we just end up loading it immediately a couple of lines later. If that's preferable that's fine.